test/Microsoft.AspNetCore.Razor.Tools.Test/Infrastructure/ToolTestFixture.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Xml.Linq;
+using Microsoft.DotNet.Cli.Utils;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Tools.Test.Infrastructure
+{
+    public class ToolTestFixture : IDisposable
+    {
+        private readonly string _defaultNugetPackageLocation;
+
+        public ToolTestFixture()
+        {
+            PackSrcDirectory();
+
+            _defaultNugetPackageLocation = Environment.GetEnvironmentVariable("NUGET_PACKAGES");
+            Environment.SetEnvironmentVariable("NUGET_PACKAGES", TestArtifactsTempPackagesDirectory);
+
+            InitializeTestApps();
+        }
+
+        public string RootDirectory => Path.GetFullPath(Path.Combine("..", ".."));
+
+        public string SrcDirectory => Path.Combine(RootDirectory, "src");
+
+        public string TestArtifactsDirectory => Path.Combine(RootDirectory, "testartifacts");
+
+        public string TestArtifactsTempPackagesDirectory => Path.Combine(TestArtifactsDirectory, "packages");
+
+        public string TestAppDirectory => Path.Combine(RootDirectory, "testapps");
+
+        public StringBuilder DotNet(string projectDirectory, string commandName, params string[] args)
+        {
+            var command = Command.CreateDotNet(commandName, args);
+            var outputWriter = new StringWriter();
+
+            var exitCode = command
+                .ForwardStdErr(outputWriter)
+                .ForwardStdOut(outputWriter)
+                .WorkingDirectory(projectDirectory)
+                .Execute()
+                .ExitCode;
+
+            var outputBuilder = outputWriter.GetStringBuilder();
```

Comment:
`var outputString = outputWriter.ToString();` and have this method return a `string` instead.